doc/thing/supported-boards/knot-dongle.rst

```diff
@@ -0,0 +1,125 @@
+KNoT Dongle
+===========
+
+Overview
+--------
+
+The KNoT Dongle (nRF52840-PCA10059) is a low-cost board manufactured by Nordic
+Semiconductor and it has a user programmable RGB LED.
```

Comment:
has **an** user

Reply:
I think it should be "a user" since it is related to how the word sounds.

Reply:
Let's just pretend it didn't happen :D